web/themes/custom/badcamp2017/sass/styles.scss

```diff
@@ -1,9 +1,45 @@
-@import url('https://fonts.googleapis.com/css?family=Roboto+Slab');
+@import url('https://fonts.googleapis.com/css?family=Roboto+Slab|Roboto');
+@import "../node_modules/foundation-sites/scss/foundation";
+@import "variables";
+@include foundation-everything();
 
-@import "../node_modules/susy/sass/susy";
-@import "../node_modules/breakpoint-sass/stylesheets/breakpoint";
+@import "components/header";
+@import "components/featured";
 
-@import "variables/variables";
-@import "base/base";
-@import "layout/layout";
-@import "components/components";
+
+/**
+ * NB: This was left for now, as it was just a copy of how the sponsors block was
+ * themed on the splash page.  It will be replaced during the design implementation.
+ */
+.core-sponsors-block {
+  margin: 1em 0;
+  padding-top: 55px;
+  border-top: 1px solid black;
+  .rows {
+    text-align:center;
+  }
+  .sponsor {
+    display:inline-block;
+    margin-right: 8px;
+  }
+  .Acquia, .Pantheon {
```

Comment:
if Acquia and Pantheon aren't our core sponsors next year, it might be better to make these generic names (.sponsor__logo, maybe).